AdvancedChangedFiles/Experience.jsx

```diff
@@ -0,0 +1,223 @@
+            isAdding: false,
+            newExperience: {
+                company: experience.company,
+                position: experience.position,
+                responsibilities: experience.responsibilities,
+                startDate: experience.startDate,
+                endDate: experience.endDate,
+            },
+        });
+    }
+
+    handleDeleteClick(id, event) {
+        event.preventDefault();
+        const updatedExperiences = this.state.experiences.filter(exp => exp.id !== id);
+        this.setState({ experiences: updatedExperiences });
+    }
+
+    handleSaveClick(event) {
+        event.preventDefault();
+        const { isEditing, newExperience, experiences } = this.state;
+
+        // Validate all required fields
+        if (
+            newExperience.company.trim() === '' ||
+            newExperience.position.trim() === '' ||
+            newExperience.responsibilities.trim() === '' ||
+            newExperience.startDate.trim() === '' ||
+            newExperience.endDate.trim() === ''
+        ) {
+            TalentUtil.notification.show("Please fill in all required fields!", "error", null, null);
+            return;
+        }
+
+        if (isEditing) {
+            const updatedExperiences = experiences.map(exp =>
+                exp.id === isEditing ? Object.assign({}, exp, newExperience) : exp
+            );
+            this.setState({
+                experiences: updatedExperiences,
+                isEditing: null,
+                newExperience: { company: '', position: '', responsibilities: '', startDate: '', endDate: '' },
+            }, () => {
+                TalentUtil.notification.show('Profile updated successfully!', 'success');
+            });
+        } else {
+            const newExperienceEntry = Object.assign({ id: Date.now() }, newExperience);
+            const updatedExperiences = experiences.concat(newExperienceEntry);
+            this.setState({
+                experiences: updatedExperiences,
+                isAdding: false,
+                newExperience: { company: '', position: '', responsibilities: '', startDate: '', endDate: '' },
+            }, () => {
+                TalentUtil.notification.show('Profile updated successfully!', 'success');
+            });
+        }
+    }
+
+    handleChange(e) {
+        const { name, value } = e.target;
+        this.setState(prevState => {
+            const newExperience = Object.assign({}, prevState.newExperience);
+            newExperience[name] = value;
+            return { newExperience };
+        });
+    }
+
+    render() {
+        const { experiences, isAdding, isEditing, newExperience } = this.state;
+
+        return (
+            <div style={{ display: 'flex', flexDirection: 'column' }}>
+                <div style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center', marginBottom: '10px' }}>
+                    <table className="ui table" style={{ flex: 1, border: 'none' }}>
```

Comment:
I can see multiple instances of these stylings on different pages. They are fine, but in an actual project, it will be better if you have:
1. Styling file and use that to this specific element
2. Use the third party library for global styling

This way, you don't have to set them every time